### DISCUSSION TITLE
`goto_word` behaviour in select mode

### DISCUSSION THREAD

**Discussion by bogdanataffirm** [helix-editor/helix#11764] [General]
### Summary

I have remapped the `goto_word` command to a single key like this:
```
[keys.normal]
h = "goto_word"

[keys.select]
h = "goto_word"
```

I can use both the default `gw` and the `h` remap in both normal and select modes.

The problem is that in select mode, using the `h` remap doesn't select everything until the target word, as I would expect. The default `gw` map behaves as expected.

### Reproduction Steps

(included in the summary)

### Platform

macOS

### Terminal Emulator

wezterm 20240203-110809-5046fc22

### Installation Method

brew

### Helix Version

helix 24.7 (079f5442)

**Comment by sicher** [comment]
Use `extend_to_word`.
